index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Theophilus Doku</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <!-- Header -->
+    <header class="header-wrapper">
```

Comment:
- [ ] You did nice work on the toolbar of your page! 💯 However, there's still some room to improve the design, one of the main goals of this project is that you demonstrate the ability to follow as closely as possible a given design, so please stick as much as possible to the Figma template 🎨. Your logo doesn't have the correct color `#6070FF` or the correct font size `18px`, and the hamburger logo is too large, please consider reducing its size! 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/211861503-acce2fd9-b670-44f6-9aa7-4180b5350651.png) | ![image](https://user-images.githubusercontent.com/77069011/211861436-40ffc5fd-30c9-4c6e-ab44-6ba32606271e.png) |